cmd/package.go

```diff
@@ -353,11 +359,20 @@ func collectDirectoryContents(packageDir string) (map[string]string, error) {
 	return contents, err
 }
 
-func pathIgnored(path string) bool {
-	return path == "" ||
+func pathIgnored(path string, capstanignore gitignore.IgnoreMatcher, isDir bool, verbose bool) bool {
+	if path == "" ||
```

Comment:
A helper should be used for this. Something that would read the capstanignore file, extend it with the patterns we are checking explicitly here and have just one matcher. Since the library does not expose an API to add new patterns, we could read the file and append these as new lines at the bottom.